### PR TITLE
Update description for BusyOnBusyEnabledType

### PR DESCRIPTION
**Pull request by xinawang**
As userOverride option is going to be rolled out in general, updating the description for the parameter

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ Accept wildcard characters: False
 ```
 
 ### -BusyOnBusyEnabledType
-Setting this parameter lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. Valid options are: Enabled, Unanswered, and Disabled. When set to Enabled, new or incoming calls will be rejected with a busy signal. When set to Unanswered, the user's unanswered settings will take effect, such as routing to voicemail or forwarding to another user. Note: The UserOverride option value is not available for use currently, if set it will be read as setting the value to Disabled.
+Setting this parameter lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. Valid options are: Enabled, Unanswered Disabled, UserOverride. When set to Enabled, new or incoming calls will be rejected with a busy signal. When set to Unanswered, the user's unanswered settings will take effect, such as routing to voicemail or forwarding to another user. Set this to UserOverride to allow users to directly set their busy options from Calls settings in Teams app.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsCallingPolicy.md:264]:
```suggestion
Setting this parameter lets you configure how incoming calls are handled when a user is already in a call or conference or has a call placed on hold. Valid options are: Enabled, Unanswered, Disabled, and UserOverride. When set to Enabled, new or incoming calls will be rejected with a busy signal. When set to Unanswered, the user's unanswered settings will take effect, such as routing to voicemail or forwarding to another user. When set to UserOverride, users can set their busy options directly from call settings in Teams app. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @xinawang, kindly commit the suggested changes. Naby thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by xinawang** [comment]
@tseward can I get a sign off? what else is needed to merge?

**Comment by yogkumgit** [comment]
@tiburd Please merge this Pr. Thanks

**Pull request merged**